### PR TITLE
Trim email input before validating

### LINKED ISSUE #385: When I copy an email from gmail, the site tells me "Must be a valid email"
Steps to reproduce:

- Go to the search box in gmail and search for any contact's 
- Copy and paste that email in the scanning box on our website
- It tells me "Must be a valid email" although the email is correct

### PR DESCRIPTION
**Pull request by nhnt11**
Fixes #385

### CODE REVIEW

**Comment by nhnt11** [comment]
My editor automatically removed trailing whitespace, let me know if you're cool with landing that in this PR or if you'd like me to revert.

File: public/js/monitor.js [hunk 1]
```diff
@@ -317,6 +317,7 @@ function handleFormSubmits(formEvent) {
     return;
   }
   const thisForm = formEvent.target;
+  thisForm.email.value = thisForm.email.value.trim();
```
Comment (groovecoder) [inline public/js/monitor.js]: We should assign this to an `emailValue` variable now that we're using it a few times:

```js
const emailValue = thisForm.email.value.trim();
if (!emailValue || !isValidEmail(emailValue))
```

**Comment by groovecoder** [comment]
Closing this PR since it now includes the change from #396 that changes functionality that mis-matches our "Your email will not be stored." copy. @nhnt11 - can you open a new PR that has just the trimming functionality?

**Comment by pdehaan** [comment]
> My editor automatically removed trailing whitespace, let me know if you're cool with landing that in this PR or if you'd like me to revert.

:+1: Submitted PR #403 to add ESLint rules for banning trailing whitespace to avoid diff churn.

**Pull request closed without merging**